packages/node_modules/@webex/internal-plugin-conversation/src/encryption-transforms.js

```diff
@@ -207,6 +209,11 @@ export const transforms = toArray('outbound', {
     alias: 'encryptVerbActivityWithKey'
   },
 
+  encryptCardactionActivity: {
```

Comment:
maybe im way off base here but CardAction <- capital A makes more sense to me.

Reply:
The capitalize function in encryptObject will capitalize cardAction -> Cardaction.

Reply:
https://github.com/webex/spark-js-sdk/blob/7c958dba34d63ec903921e70ec8d274cbc7780d9/packages/node_modules/%40webex/internal-plugin-conversation/src/encryption-transforms.js#L40